### ISSUE TITLE
Home page broken images

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ocw-studio#1044]
### Steps to Reproduce

- visit https://ocwnext.odl.mit.edu/ or https://ocw-preview.odl.mit.edu/ or https://ocw-published.odl.mit.edu/ 

### Expected Behavior

- images added through studio should publish, for example, testimonials and promo images. 

### Actual Behavior

images are returning 404, for example: 
- https://ocw-published.odl.mit.edu/https:/ol-ocw-studio-app-production.s3.amazonaws.com/ocw-www/412264223fc744c8baa7e3f74755f489_blackburn_th.jpg
- https://ocw-published.odl.mit.edu/https:/ol-ocw-studio-app-production.s3.amazonaws.com/ocw-www/b2c1e39b133349be99332b11c5640078_chalk-radio.png

**Comment by gumaerc** [comment]
Adding some further details here, we found that this issue was caused by running `sync_backend_to_db` on `ocw-www`, then subsequently running `sync_website_to_backend`.  The S3 URL prefix is prepended to file URLs during syncing to the backend, then the same thing seemingly happens when syncing in the other direction, doubling up the domain at the front of the URL.  The immediate issue was fixed manually in a Django shell, but `sync_backend_to_db` still needs to be looked at to prevent this issue from happening in the future.
